### PR TITLE
210 bug fix edit price

### LINKED ISSUE #210: Bug: invalid inputs in edit price command
- Able to add special characters in price
- able to have price more than 2 d.p.
- price printed not in 2 d.p.

### PR DESCRIPTION
**Pull request by ziyi105**
Fixes #210

### CODE REVIEW

File: src/main/java/seedu/cafectrl/parser/Parser.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public class Parser implements ParserUtil {
     /** The rest of Command Handler Patterns*/
     private static final String LIST_INGREDIENTS_ARGUMENT_STRING = "(\\d+)";
     private static final String DELETE_ARGUMENT_STRING = "(\\d+)";
-    private static final String EDIT_PRICE_ARGUMENT_STRING = "index/(\\d+) price/(\\d+(\\.\\d+)?)";
+    private static final String EDIT_PRICE_ARGUMENT_STRING = "index/(.*) price/(.*)";
```
Comment (DextheChik3n) [inline src/main/java/seedu/cafectrl/parser/Parser.java:73]: I noticed that this regex still accepts more than 2 d.p. numbers,
image

I remember you said you plan to round it up to 2 d.p. , can I check if this was implemented in this PR?  if not, another suggestion I can give is to use this regex (inspired from my add dish regex) which can reject inputs more than 2 d.p. but can allow the user to enter prices up to 2 d.p. (e.g. price/2 OR price/2.0 OR price/2.01 OR price/.01)

`"index/([0-9]*) price/([0-9]*\\.[0-9]{0,2}|[0-9]+)$"`

**Review by DextheChik3n — COMMENTED** [review]
Added comment on input handling

File: src/main/java/seedu/cafectrl/parser/Parser.java (hunk 1 above)
Comment (DextheChik3n) [reply src/main/java/seedu/cafectrl/parser/Parser.java:73]: If you do use my regex remember to do extensive testing :P

Comment (ziyi105) [reply src/main/java/seedu/cafectrl/parser/Parser.java:73]: I implemented the round up feature in dish feature, so no matter how many dps are provided by the user the app will automatically update to 2dp

**Review by DextheChik3n — APPROVED** [review]
All's lookin good to me 👍

**Pull request merged**